framework/src/main/java/org/checkerframework/common/basetype/BaseTypeChecker.java

```diff
@@ -599,6 +600,8 @@ private void printStoredMessages(CompilationUnitTree unit) {
         final String message;
         /** The source code that the message is about. */
         final @InternedDistinct Tree source;
+        /** Stores the stack trace till the point the checker encounters an error. */
+        final StackTraceElement @Nullable [] trace;
```

Comment:
Why is the field `@Nullable`? Same for the `trace` parameter that sets the field.
`Thread.currentThread().getStackTrace()` returns something non-null.

Reply:
I think the IDE added it automatically when I initialized it. I have removed the annotation since it was, as you said, redundant.